typescript/infra/src/govern/multisend.ts

```diff
@@ -51,35 +56,72 @@ export class SafeMultiSend extends MultiSend {
   }
 
   async sendTransactions(calls: CallData[]) {
-    const safeSdk = await getSafe(
-      this.chain,
-      this.multiProvider,
-      this.safeAddress,
-    );
-    const safeService = getSafeService(this.chain, this.multiProvider);
-
-    const safeTransactionData = calls.map((call) => {
-      return {
-        to: call.to,
-        data: call.data.toString(),
-        value: call.value?.toString() || '0',
-      };
+    const { safeSdk, safeService } = await getSafeAndService({
+      chain: this.chain,
+      multiProvider: this.multiProvider,
+      safeAddress: this.safeAddress,
     });
-    const nextNonce = await safeService.getNextNonce(this.safeAddress);
-    const safeTransaction = await safeSdk.createTransaction({
+
+    if (isZeroishAddress(safeSdk.getMultiSendAddress())) {
+      console.log(
+        `MultiSend contract not deployed on ${this.chain}. Proposing transactions individually.`,
+      );
+      await this.proposeIndividualTransactions(calls, safeSdk, safeService);
+    } else {
+      await this.proposeMultiSendTransaction(calls, safeSdk, safeService);
+    }
+  }
+
+  // Helper function to propose individual transactions
+  private async proposeIndividualTransactions(
+    calls: CallData[],
+    safeSdk: any,
+    safeService: any,
+  ) {
+    for (const call of calls) {
+      const safeTransactionData = createSafeTransactionData({ call });
+      const safeTransaction = await createSafeTransaction({
+        safeSdk,
+        safeService,
+        safeAddress: this.safeAddress,
+        safeTransactionData,
+      });
+      await this.proposeSafeTransaction(safeSdk, safeService, safeTransaction);
+    }
+  }
+
+  // Helper function to propose a multi-send transaction
+  private async proposeMultiSendTransaction(
+    calls: CallData[],
+    safeSdk: any,
```

Comment:
nit: do these need to be `any`? seems like they come from some fns that have types